addons/mail/static/src/new/rtc/call_invite_request_popup.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.call_invite_request_popup" owl="1">
+        <div class="o_CallInviteRequestPopup d-flex flex-column m-2 p-5 border border-dark rounded-1 bg-900" t-attf-class="{{ className }}" t-ref="root">
+            <div t-if="props.thread.rtcInvitingSession" class="o_CallInviteRequestPopup_partnerInfo d-flex flex-column justify-content-around align-items-center text-nowrap">
+                <img class="o_CallInviteRequestPopup_partnerInfoImage mb-2 rounded-circle cursor-pointer"
+                    t-att-src="props.thread.rtcInvitingSession?.channelMember?.persona.avatarUrl"
+                    t-on-click="onClickAvatar"
+                    alt="Avatar"/>
+                <span class="o_CallInviteRequestPopup_partnerInfoName w-100 fw-bolder text-truncate text-center overflow-hidden" t-esc="props.thread.rtcInvitingSession.channelMember.persona.name"/>
+                <span class="o_CallInviteRequestPopup_partnerInfoText fst-italic opacity-75">Incoming Call...</span>
+            </div>
```

Comment:
I managed to have an invitation displaying strangely (no image or name)

![image](https://user-images.githubusercontent.com/38213504/216650521-ba16eab6-e2de-4a8d-9a9a-6d3d887667df.png)